python/pyspark/ml/classification.py

```diff
@@ -500,16 +500,12 @@ def featureImportances(self):
         """
         Estimate of the importance of each feature.
 
-        This generalizes the idea of "Gini" importance to other losses,
-        following the explanation of Gini importance from "Random Forests" documentation
-        by Leo Breiman and Adele Cutler, and following the implementation from scikit-learn.
+        Each feature's importance is the average of its importance across all trees in the ensemble
```

Comment:
There was some discussion on this [here](https://github.com/apache/spark/pull/11961). Basically, since the random forest and GBT importance is just an average of the importances for single trees, we can just state that here and link to the doc for single trees, which explains how those are computed. Otherwise, we copy/paste the same explanation 6 times.